pkg/controller/configfile/configfile.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package configfile
+
+import (
+	"fmt"
+
+	config "github.com/cert-manager/cert-manager/internal/apis/config/controller"
+)
+
+type Any = interface{}
```

Comment:
I'm a little wary of empty interfaces. Maybe we could use runtime.Object here, would that work?